runner/operator/access/v1_0_0/structural_variants/__init__.py

```diff
@@ -104,7 +105,7 @@ def construct_sample_inputs(self, tumor_sample_id, tumor_bam):
             tumor_sample_names = [tumor_sample_id]
             tumor_bams = [{
                 "class": "File",
-                "location": 'juno://' + tumor_bam['path']
+                "location": tumor_bam['location']
             }]
```

Comment:
I feel like this could be a bit cleaner, for example maybe instead of re-building the file objects to put in the inputs, I could just re-use the Files directly?

Or even better, maybe there is a cleaner way to apply the port objects directly in the inputs